test/e2e/common.go

```diff
@@ -47,10 +48,85 @@ func createNamespace(ctx context.Context, client testClient, namespace string) e
 	return err
 }
 
+// createNamespaceAndRbac creates a kubernetes namespace, service account, clusterrole, and clusterrolebinding
```

Comment:
Let's break this into separate functions.  This is not very re-usable, everything needs to be done together.